src/generate/interrupt.rs

```diff
@@ -161,9 +156,7 @@ pub fn render(
         unsafe impl bare_metal::Nr for Interrupt {
```

Comment:
```suggestion
         #[repr(u8)]
         unsafe impl bare_metal::Nr for Interrupt {
```

Reply:
I suppose this should go on the enum and not on the impl?

Reply:
yes